Reflection_Engine/Query/AllEvents.cs

```diff
@@ -38,18 +38,6 @@ public static List<Event> AllEvents()
             Log log = Query.DebugLog();
             return log.AllEvents;
         }
-
-
-        /***************************************************/
```

Comment:
Same thing for here. I do prefer to keep the two methods in a single file called `Events`. It makes the code more readable to have them together on a file and it would be a bit silly to have to use an enum as argument just so we can have a matching method name.